pkg/apis/options/legacy_options.go

```diff
@@ -50,7 +60,9 @@ type LegacyUpstreams struct {
 func legacyUpstreamsFlagSet() *pflag.FlagSet {
 	flagSet := pflag.NewFlagSet("upstreams", pflag.ExitOnError)
 
-	flagSet.Duration("flush-interval", time.Duration(1)*time.Second, "period between response flushing when streaming responses")
+	flagSet.Duration("flush-interval", defaultFlushInterval, "period between response flushing when streaming responses")
+	flagSet.Duration("timeout", defaultTimeout, "dialer timeout")
```

Comment:
Be sure to get these documented in here as well `https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/configuration/configuration.md`

Can you also make the description include more details?

Reply:
yes and yes!